lib/req/test.ex

```diff
@@ -645,4 +645,48 @@ defmodule Req.Test do
         """
     end
   end
+
+  ## Plug Parser
+
+  @doc false
+  def read_request_body(conn, opts) do
```

Comment:
people might be doing `import Req.Test`, wdyt about preventing this from being imported?

```suggestion
  def __read_request_body__(conn, opts) do
```